kernel/build.sh

```diff
@@ -27,6 +27,8 @@ while (( ${#} )); do
                     "AArch64") TARGETS=( "${TARGETS[@]}" "aarch64-linux-gnu" ) ;;
                     "ARM") TARGETS=( "${TARGETS[@]}" "arm-linux-gnueabi" ) ;;
                     "PowerPC") TARGETS=( "${TARGETS[@]}" "powerpc-linux-gnu" "powerpc64-linux-gnu" "powerpc64le-linux-gnu" ) ;;
+                    # SystemZ is consumed until Linux 5.6
```

Comment:
Should this also wait until then?

Reply:
We can build a mainline kernel just fine with latest tip of tree. I just do not really want to add a prerelease kernel to this script (but we could when 5.6-rc1 drops). I do not have a strong opinion.